lib/Optimizer/GraphOptimizer/PassManager.cpp

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Optimizer/GraphOptimizer/PassManager.h"
+
+#include "glow/Graph/Graph.h"
+#include "glow/Optimizer/GraphOptimizer/FunctionPasses.h"
+
+#include "llvm/Support/CommandLine.h"
+
+#include <glog/logging.h>
+
+using namespace glow;
+
+namespace {
+llvm::cl::OptionCategory passManagerCat("PassManager Options");
+
+llvm::cl::opt<bool> verifyBeforeAllPassesOpt(
```

Comment:
Very nice! Could you also add similar flags for dumping the graph before/after some/all passes?

Reply:
Done.